src/fetch_package.js

```diff
@@ -173,33 +124,25 @@ const fetchPackage = function(url, cacheBehavior, requestStats,
                 // If we get here, we have a 5xx error or similar
                 // (socket timeout, maybe).  Let's retry a few times.
                 if (triesLeftAfterThisOne > 0) {
-                    fetchPackage(url, cacheBehavior, requestStats,
-                                 triesLeftAfterThisOne - 1)
+                    fetchPackage(url, requestStats, triesLeftAfterThisOne - 1)
                         .then(resolve, reject);
                     return;
                 }
 
                 // OK, I give up.
                 reject(err);
             } else {
-                const cacheableValue = {
-                    lastModified: res.header && res.header["last-modified"],
-                    script: new vm.Script(res.text, {filename: url}),
-                };
+                const script = new vm.Script(res.text, {filename: url})
                 // Pass in a size estimate; it's really just an estimate
                 // though as we don't consider anything but the script text.
                 // Since we're running in node, we assume 2 bytes
                 // per character in the text. We aren't accounting for
                 // other info associated with that, though.
-                cacheableValue.script.size = res.text.length * 2;
-
-                if (cacheBehavior !== 'ignore') {
-                    cache.set(url, cacheableValue, cacheableValue.script.size);
-                }
+                script.size = res.text.length * 2;
```

Comment:
Is this used by anything? It seems like this may have only been used by `cache.set` before?

Reply:
Oh yeah, I'll look. I think we use if for the requestStats, which I'm not sure if they're used or not. I'll check.

Reply:
Yeah, this is used to track the package size for our request states stuff, so I'll leave it as-is for now.

Reply:
I've added an extra comment though.